src/main/converter/opds.ts

```diff
@@ -176,6 +176,14 @@ export class OpdsFeedViewConverter {
             navigation = feed.Navigation.map((item) => {
                 return this.convertOpdsLinkToView(item);
             });
+
+            // concatenate all relative path to an absolute URL path
+            navigation = navigation.map((nav) => {
+                if (nav.url && !/[a-zA-Z]+\:\/\//.exec(nav.url)) {
```

Comment:
The sequence `\:` escapes a non-reserved character. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions